src/System.Private.CoreLib/shared/System/PinnableBufferCache.cs

```diff
@@ -89,6 +89,7 @@ internal object Allocate()
             if (!m_FreeList.TryPop(out returnBuffer))
                 Restock(out returnBuffer);
```

Comment:
Nit: It would be nice to delete `PINNABLEBUFFERCACHE_MSCORLIB` ifdef above. It is confusing that the define is not defined.